src/main.py

```
@@ -0,0 +1,32 @@
+
+    # execute sci-kit learn pipeline to perform preprocessing tasks
+    data_processor.preprocess_raw_data()
+    logger.info("Data preprocessed.")
+
+    # Split the data
+    X_train, X_test, y_train, y_test = data_processor.split_data()
+    logger.info(f"Data split into training and test sets.")
+    logger.debug(f"Training set shape: {X_train.shape}, Test set shape: {X_test.shape}")
+
+if __name__ == '__main__':
+    run()
```

Ruff

No newline at end of file

Add trailing newline

(W292)